docker/images/node/Dockerfile

```diff
@@ -2,7 +2,19 @@ FROM node:slim
 
 WORKDIR /usr/app/src/
 
+ENV DEPS_DIR=/usr/app/refarm-site
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends --no-install-suggests -y ssh git \
+    && git clone https://github.com/fidals/refarm-site.git $DEPS_DIR \
```

Comment:
The refarm-site is versioned, so we should get its version from requirements.txt or somewhere else